eq/server/config/display.cpp

```diff
@@ -49,6 +49,12 @@ void _choosePixelViewport( Window& window )
     if( width == fabric::UNDEFINED && height == fabric::UNDEFINED )
     {
         window.setViewport( Viewport( .25f, .2f, .5f, .5f ));
+
+        // Make the window aspect ratio 16:10, assuming square pixels
```

Comment:
For example two full HD screens:

w = 540 * 1.6 = 864
x = 432

-> window is in the middle of the first screen

before: (3840x1080)

x = 960
w = 1920

-> window is 32x10 AR, between the two screens.

Where do you think this should be documented? Imo with this code?